src/docs/sdk/event-payloads/properties/transaction.mdx

```diff
@@ -0,0 +1,9 @@
+`transaction`: 
+
+: _Recommended_. In case of a root span, this property is recommended. Set the `transaction` to give your tree of spans a dedicated name. If it's empty we fall back to `<unlabeled transaction>`.
```

Comment:
Is it valid for non-root spans? Is there a defined behavior in that case?

Like for `op`, I feel the exact behavior in Relay is an implementation detail, I'd leave it out. Your call.